packages/belos/src/BelosOutputManager.hpp

```diff
@@ -160,7 +161,7 @@ namespace Belos {
     // Initialize MPI
     int mpiStarted = 0;
     MPI_Initialized(&mpiStarted);
-    if (mpiStarted) MPI_Comm_rank(MPI_COMM_WORLD, &MyPID);
+    if (mpiStarted) MPI_Comm_rank(get_global_comm(), &MyPID);
```

Comment:
Since Belos has hard dependency on Teuchos, we could use Teuchos::DefaultComm::getComm() here (which is a wrapper for MPI_COMM_WORLD) seeing that there's only a single occurence in this package. What do you think?